execution_engine_testing/test_support/src/wasm_test_builder.rs

```diff
@@ -1171,27 +1171,38 @@ where
     }
 
     /// Advances eras by num_eras
-    pub fn advance_eras_by(&mut self, num_eras: u64, reward_items: Vec<RewardItem>) {
+    pub fn advance_eras_by(
+        &mut self,
+        num_eras: u64,
+        reward_items: impl IntoIterator<Item = RewardItem>,
+    ) {
+        let step_request_builder = StepRequestBuilder::new()
+            .with_protocol_version(ProtocolVersion::V1_0_0)
+            .with_reward_items(reward_items)
+            .with_run_auction(true);
+
         for _ in 0..num_eras {
-            let step_request_builder = StepRequestBuilder::new()
+            let step_request = step_request_builder
+                .clone()
```

Comment:
This probably doesn't improve the performance and makes it less readable IMO but let's not change it in this PR, it's already been too long.